modules/agent/pkg/controllers/cluster/cluster.go

```diff
@@ -33,14 +33,7 @@ type handler struct {
 	reported         fleet.AgentStatus
 }
 
-func Register(ctx context.Context,
-	agentNamespace string,
-	clusterNamespace string,
-	clusterName string,
-	checkinInterval time.Duration,
-	nodes corecontrollers.NodeCache,
-	clusters fleetcontrollers.ClusterClient) {
-
+func Register(ctx context.Context, agentNamespace string, clusterNamespace string, clusterName string, checkinInterval time.Duration, nodes corecontrollers.NodeCache, clusters fleetcontrollers.ClusterClient) {
```

Comment:
Out of curiosity, is there any reason to change this function arguments definition?